automation/taskcluster/update_android_components.sh

```diff
@@ -52,7 +52,7 @@ fi
 echo "Pushing branch to GitHub"
 URL="https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/$GITHUB_USER/$REPO/"
 # XXX git sometimes leaks the URL including the token when the network request failed (regardless of --quiet).
-git push --force --no-verify --quiet "$URL" "$BRANCH" > /dev/null 2>&1 || echo "Failed ($?)"
```

Comment:
Question: I guess the reason why echo `Failed` is to show that an error occurred since all logs are sent to `/dev/null`, right?

Reply:
As far as I can tell, yes.